controller.go

```diff
@@ -70,8 +71,13 @@ func (c *Controller) CreateConnection(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	log.Println("id: ", msg.DeviceId)
-	reply := c.sigServer.NewConnection(msg)
-	replyJSON(w, reply)
+	reply, err := c.sigServer.NewConnection(msg)
+	if err != nil {
+		log.Println("Failed to send message to device: ", err)
```

Comment:
Done
A status code still needs to be returned from each signaling server endpoint because the host orchestrator's reply includes one. IOW If the host orchestrator returns a non-2xx status code and a JSON object, both need to be returned to the client.